docs/Installing.asciidoc

```diff
@@ -63,7 +63,21 @@ single-instance container. You can spawn a new shell via:
 podman exec -ti openqa /bin/bash
 ----
 
-From there, you can trigger a new job or clone an existing one, e.g.:
+From there, you can schedule or clone jobs. If you wish to test openSUSE, you have to clone the https://github.com/os-autoinst/os-autoinst-distri-opensuse[openSUSE test data]. This can be done easily by running a script, which will clone both test files and needles:
+
+[source,sh]
+----
+/usr/share/openqa/script/fetchneedles
+----
+
+Afterwards you can clone an existing job, e.g.:
+
+[source,sh]
+----
+openqa-clone-job https://openqa.opensuse.org/tests/5107001
```

Comment:
If you refer to a osado test here, then this won't work without manual cloning.
So removing the mention of fetchneedles will result in this example not working.